makefile.shade

```diff
@@ -45,6 +45,9 @@ functions
         static string repositoryInclude = GetEnvironmentParameter("KOREBUILD_REPOSITORY_INCLUDE");
         static string repositoryExclude = GetEnvironmentParameter("KOREBUILD_REPOSITORY_EXCLUDE");
         
+        static string gateBranch = GetEnvironmentParameter("KOREBUILD_GATE_BRANCH");
```

Comment:
There's a bunch of variable env names right above this line which start with that name though?